packages/scaffolding-cli/templates/build/azDevops/azure/azure-pipeline-infrastructure-aks.yml

```diff
@@ -67,169 +67,336 @@ variables:
+              # TODO: make this a complex object so that additional args can be passed in
+              - $(Agent.BuildDirectory)/s/$(self_repo)/build/k8s_manifests/aks/nginx-ingress.yml
+            external_dns_yml: $(Agent.BuildDirectory)/s/$(self_repo)/build/k8s_manifests/aks/external-dns.yml
+            domain_filter: $(base_domain_nonprod)
+
+  - stage: prod
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    variables:
+    # You can find notes in the READMEs around which values to use for each ENV variable group
+    - group: amido-stacks-infra-credentials-prod
+    jobs:
+    - job: InfraProd
+      variables:
+        ARM_SUBSCRIPTION_ID: $(azure_subscription_id)
+        ARM_CLIENT_ID: $(azure_client_id)
+        ARM_CLIENT_SECRET: $(azure_client_secret)
+        ARM_TENANT_ID: $(azure_tenant_id)
+      pool:
+        vmImage: ubuntu-18.04
+      container: terraform_custom
+      steps:
+        - checkout: self
+
+        - checkout: templates
+          persistCredentials: true
+
+        - bash: |
+            echo '##vso[build.updatebuildnumber]$(version_build_number)'
+          displayName: 'Update: Build Number'
+
+        # Create TF infra
+        - template: azDevOps/azure/templates/v2/steps/deploy-terraform-resource.yml@templates
+          parameters:
+            # Terraform Basic Config
+            terraform_working_dir: '$(Agent.BuildDirectory)/s/$(self_repo)/$(self_repo_tf_src)'
+            # changing this would require changing the downstream consumers of these outputs
+            terraform_output_artefact: '  tfoutputs'
+            # Terraform State Config
+            terraform_backend_client_id: $(ARM_CLIENT_ID)
+            terraform_backend_client_secret: $(ARM_CLIENT_SECRET)
+            terraform_backend_tenant_id: $(ARM_TENANT_ID)
+            terraform_backend_subscription_id: $(ARM_SUBSCRIPTION_ID)
+            terraform_state_rg: $(tf_state_rg)
+            terraform_state_storage: $(tf_state_storage)
+            terraform_state_container: $(tf_state_container)
+            terraform_state_key: $(tf_state_key)
+            terraform_state_workspace: prod
+            # Global Config
+            company: 'amido'
+            project: 'stacks'
+            environment: ''
+            azure_cli_providers: [
+              { name: MSIPreview, namespace: Microsoft.ContainerService }
+            ]
+            # Azure config
+            tags: ''
+            # Terraform Resource Specific Config
+            terraform_extra_properties: {
+              TF_VAR_client_secret: $(azure_client_secret),
+              TF_VAR_tenant_id: $(azure_tenant_id),
+              TF_VAR_resource_group_location: northeurope,
+              TF_VAR_name_company: $(company),
+              TF_VAR_name_project: $(project),
+              TF_VAR_name_component: $(domain),
+              TF_VAR_name_environment: prod,
+              TF_VAR_create_acr: true,
```

Comment:
you probably want to set this to false - I think there is a param to pass in the ACR name

Reply:
Don't we want a Prod ACR, then you have a promotion stage and deploy to prod on the Apps (like .NET has for Cycle 1)

Reply:
That way we also give them the code to promote images between ACRs as you probably will have to do anyway when moving between NonProd and Prod

Reply:
Because NonProd and Prod probably will be in separate subscriptions in a real project.

Reply:
@dnitsch Let's roll with this for now and we'll improve later? Opened a ticket here: https://github.com/amido/stacks-terraform/issues/4